pyproject.toml

```diff
@@ -19,13 +19,13 @@ packages = [
 
 [tool.poetry.dependencies]
 python = "3.9.6"
```

Comment:
can we update it to `python = "^3.9"`? do you remember any requirements for the minimum `3.9.6` version?>

Reply:
Not in scope of this PR. Changed in #123